transports/webrtc-utils/Cargo.toml

```
@@ -0,0 +1,38 @@
+[package]
+edition = "2021"
+license = "MIT"
+name = "libp2p-webrtc-utils"
+rust-version.workspace = true
```

```suggestion
rust-version = { workspace = true }
```

Nit pick to be consistent across the workspace. Allows for easy search and replace.

---

Ah yes I see, fixed.